content/docs/product-tour.mdx

```diff
@@ -14,94 +14,94 @@ Here we give a quick **2 minute overview of TinaCMS**, and how it fits into your
 
 TinaCMS is an open-source Content Management System (CMS) that seamlessly integrates with your Markdown workflow.
 
-![Intro-gif](https://res.cloudinary.com/forestry-demo/image/upload/v1645712511/tina-io/docs/your-blocks.gif)
+<WebmEmbed embedSrc="/img/docs/your-blocks.webm"/>
```

Comment:
**question**: There is an image being shown where there wasn't 1 previously - is this intentional?

![image](https://github.com/user-attachments/assets/b4eb9648-ebf6-4924-a078-d93f4e4cfa40)
**Figure: New image on the updated page at left, which also appears to be shown in the gutter image**

Reply:
Resolved